src/test/java/com/google/javascript/cl2dts/types.js

```diff
@@ -11,3 +11,21 @@ types.c = 's';
 
 /** @type {Object} */
 types.d = {};
+
+/** @type {Array<?>} */
```

Comment:
Yes, is it worth it? I suppose users will frequently be exposed to typescript idioms by reading the typings we expose through their editor and error messages, so maybe we want to teach them